src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw

```diff
@@ -753,11 +753,11 @@
     <comment>Text on the button that allows the user to clear the stored key.</comment>
   </data>
   <data name="AISettings_OpenAIDescription" xml:space="preserve">
-    <value>OpenAI is provided by a third-party and not Microsoft.  When you send a message in Terminal Chat, your chat history and the name of your active shell are sent to the third-party AI service for use by OpenAI. {0}.  Your use of OpenAI is governed by the relevant third-party terms, conditions, and privacy statement. </value>
+    <value>OpenAI is provided by a third-party and not Microsoft.  When you send a message in Terminal Chat, your chat history and the name of your active shell are sent to the third-party AI service for use by OpenAI. {0}. Your use of OpenAI is governed by the relevant third-party terms, conditions, and privacy statement. </value>

```

Comment:
Not sure why this diff is showing here when the target branch has this same commit...